x/validator/handler_test.go

```diff
@@ -51,11 +53,18 @@ func Setup(t *testing.T) TestSetup {
 	)
 	dclauthK.SetAccount(ctx, account)
 
+	validator1, _ := types.NewValidator(
+		sdk.ValAddress(testconstants.ValidatorAddress1),
+		testconstants.ValidatorPubKey1,
+		types.Description{Moniker: "Validator 1"},
+	)
+	k.SetValidator(ctx, validator1)
```

Comment:
It will be more consistent to also create the node admin of this validator (before the validator creation).